### PR TITLE
Improve fabric.IToSVGOptions dimensions types

### PR DESCRIPTION
**Pull request by soywod**
The `IToSVGOptions` dimensions (`width` and `height`) should also accept `string`, since they can contain units.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/fabricjs/fabric.js/blob/d61a34347af14a2f1d48831056308cb2abb14cc8/src/static_canvas.class.js#L1332-L1333
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/fabric/fabric-impl.d.ts [hunk 1]
```diff
@@ -191,11 +203,11 @@ interface IToSVGOptions {
     /**
      * desired width of svg with or without units
      */
-    width?: number;
+    width?: number | string;
```
Comment (soywod) [inline types/fabric/fabric-impl.d.ts:206]: Real changes here (the rest is from Prettier).

File: types/fabric/fabric-impl.d.ts [hunk 2]
```diff
@@ -191,11 +203,11 @@ interface IToSVGOptions {
...
     /**
      * desired height of svg with or without units
      */
-    height?: number;
+    height?: number | string;
```
Comment (soywod) [inline types/fabric/fabric-impl.d.ts:210]: Real changes here (the rest is from Prettier).

**Review by BradleyHill — APPROVED** [review]
Didn't map width/height directly to fabric.js but seems most svg width/height in fabric.js passed to html so string would be valid.
Prettier gonna have to trust. Ostensibly what little i looked at seems legit TS. Don't have inclination to walk the file. Trust fall.

**Comment by elibarzilay** [comment]
@soywod: is this ready for merging?

**Comment by soywod** [comment]
@elibarzilay yes ofc, do I miss sth?

**Comment by elibarzilay** [comment]
Just making sure...

**Pull request merged**